TI-reports/2024/2024-Q4-Alpha-Omega.md

```diff
@@ -0,0 +1,101 @@
+|KR 1.4: Organize quarterly roundtables for at least 5 major ecosystems  to share information, build connections, and collaborate, resulting in at least one new project or joint publication started in 2024.|On target|
+|**O2: The top 10,000 open source projects are free of critical security vulnerabilities**||
+|KR 2.1: Drive adoption of key security processes, including static analysis, credential scanning, the use of private vulnerability disclosures, structured metadata (Security Insights) and the use of multi-factor authentication by maintainers of 500 critical projects from the top 10,000 by the end of 2024.|Not Started|
+|KR 2.2: Independently scan, triage, and notify maintainers when critical vulnerabilities are found in 2,000 projects, chosen from the top 10,000 by the end of June 2024, with emphasis on clearing a "section of the beach" by focusing on the top PyPI packages.|On target|
+|KR 2.3: Publish in a machine readable format the attestations for all packages from 2.2 that returned no vulnerabilities and those that found vulnerabilities which were subsequently fixed and verified.|On target|
+|**O3: Enhance Alpha-Omega's effectiveness in driving security improvements through deliberate innovation and experimentation**||
+|KR 3.1: By the end of 2024, run three experiments to explore new strategies for reducing security risk within the open source ecosystems, share the results/learnings, using them to refine our overall strategy and objectives for 2025.|On Target|
+|**O4: Run an operationally efficient and effective program**||
+|KR 4.1: Allocate at least 85% of our yearly spend to activities directly in support of our mission.|On Target|
+|KR 4.2: Receive at least $5 million in renewed funding in 2024.|Completed|
+|KR 4.3: For each partner engagement, at least 70% of the objectives defined within the respective agreement are met within the defined time period.|On target|
+
+### Purpose
+
+To catalyze sustainable security improvements to the world's most critical open source projects and ecosystems.
+
+### Current Status
+
+We're active and healthy. Our team consists of Henri Yandell (Amazon Web Services), Bob Callaway (Google), and Michael Scovetta (Microsoft), 
+supported by Michael Winser (independent) and Michelle Martineau and Tracy Li from the Linux Foundation.
+
+We've received $5M in funding in 2024 and are on target to spend over $6M by the end of the year. 
+
+
+### Up Next
+
+We'll be at the Linux Foundation Member Summit next week, sponsoring a Happy Hour for OpenSSF Governing Board and TAC
+on 11/17 from 5-6 PM at the Mansion Bar & Terrace (at the Silverado Resort). We hope to see you there.
+
+We continue to hold monthly public meetings (on the OpenSSF community calendar).
+
+Our 2024 Annual Report will be published in early January 2025.
+
+Some key opportunities to engage:
+
+* Our next monthly report is due out around July 5th.
```

Comment:
It's hard to follow the timeline in this section. It's unclear if this is part of a previous update that wasn't merged or if this section is incorrect.

Reply:
Sorry @mlieberman85, copy-paste error from Q2, fixed.

Reply:
Looks good now!